### ISSUE TITLE
Log format is not correct

### ISSUE THREAD

**Issue by longdv** [anvox/resque-alarm#8]
Log should be a json string, so it can be parsed by another tool, ex: logcollector

Actual:
`{:app=>"Saleshood", :source=>"resque-queue-length-alarm", :type=>"resque_queue_too_long", :queues=>{:clean_activities=>110, :background_alerts=>16}}`

Expected:
```
{"app":"Saleshood","source":"resque-queue-length-alarm","type":"resque_│
queue_too_long","queues":{"clean_activities":110,"background_alerts":16│
}}
```

**Comment by duongvanlong** [comment]
me too, same problem

**Comment by anvox** [comment]
I reverted https://github.com/anvox/resque-alarm/pull/9 on `v0.1.6` as it released. Open another to `master`

**Comment by anvox** [comment]
Is it your expectation? I dont know why was that slice?

    {"app":"Saleshood","source":"resque-queue-length-alarm","type":"resque_│
    queue_too_long","queues":{"clean_activities":110,"background_alerts":16│
    }}

**Comment by duongvanlong** [comment]
yes, because I use logtash and it can't parse json with the old format, I think it will be excellent if format is a json string
